usr/share/rear/skel/default/etc/scripts/system-setup.d/55-migrate-network-devices.sh

```diff
@@ -12,7 +12,7 @@
 #
 
 # Get the rule files (though it should be only one):
-RULE_FILES=( /etc/udev/rules.d/*persistent*{names,net}.rules )
+RULE_FILES=( /etc/udev/rules.d/*persistent*{names,net}.rules /etc/udev/rules.d/*eno-fix.rules )
```

Comment:
What is the reason that here it is only
```
... /etc/udev/rules.d/*persistent*{names,net}.rules ...
```
while in usr/share/rear/finalize/GNU/Linux/310_migrate_udev_rules.sh there is
```
... /etc/udev/rules.d/*persistent*{names,net,cd}.rules ...
```
I.e. I wonder why `/etc/udev/rules.d/*persistent*cd.rules` is not considered here.

Reply:
@jsmeix The code was original a patch coming from Heinlein Support (cfr https://github.com/rear/rear/commit/844d50b75ac4b7722f4fee7a5ee3350b93f3adb7) of 9 years ago.
 However, I agree with you that we better align both scripts.